packages/eslint-plugin-next/lib/index.js

```diff
@@ -39,12 +39,12 @@ module.exports = {
         '@next/next/next-script-for-ga': 1,
         '@next/next/no-document-import-in-page': 2,
         '@next/next/no-head-import-in-document': 2,
-        '@next/next/no-script-in-document': 2,
         '@next/next/no-script-component-in-head': 2,
         '@next/next/no-server-import-in-page': 2,
         '@next/next/no-typos': 1,
         '@next/next/no-duplicate-head': 2,
         '@next/next/inline-script-id': 2,
+        '@next/next/no-before-interactive-script-outside-document': 2,
```

Comment:
```suggestion
        '@next/next/no-before-interactive-script-outside-document': 1,
```
If we want to start with this as a warning should this be a `1` for warn instead of a `2` for error?